### ISSUE TITLE
BUG: MathJax Chinese Overlapping Issue

### ISSUE THREAD

**Issue by Bowen-0x00** [zsviczian/obsidian-excalidraw-plugin#1406]
**Your environment**
Excalidraw 1.9.26

**Describe the bug**
In LaTeX, when inputting Chinese text and formulas simultaneously, there may be an issue of overlapping between the Chinese text and the formulas.
Such as: $$你好\cfrac{1}{2}$$, looks like
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/120435922/34055798-883f-493f-8fba-1742ec0ed912)

**To Reproduce**
1. Insert a latex equation with "你好\cfrac{1}{2}"
2. See overlapping

**Expected behavior**
It should look like this
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/120435922/f43f5135-59b2-4692-9ceb-fbf77d49df3b)

**Info**
I moved the script outside the iframe and under the parent document, and it works fine. However, this affects the proper display of formulas in Obsidian Markdown.

![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/120435922/613e86d5-87d0-46ad-bf98-e6c58e86d531)

I compared the SVGs between the two scenarios and found that in the problematic case, the correct translation was not applied. I suspect that this is due to the inability to obtain font width information.
![image](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/120435922/01545054-b4da-43bf-b048-980f44b4b7d9)

**Comment by zsviczian** [comment]
yes, the basic issue is that the MathJax package that comes with Obsidian does not include the SVG package - so I need to create another instance, but MathJax can only have a single instance... thus creating the script in the iframe.
To test things out, can you try copying all the headers from Obsidian app to the iframe. Perhaps if all the style elements are there the SVG will be generated correctly also for Chinese characters. If that works, then I will copy styles while creating the iframe.

**Comment by Bowen-0x00** [comment]
It didn't work. I tried copying all the content under the <head> tag and pasting it into the <head> of the <iframe>, and then reopened the Excalidraw file.  ( I also tried copying before loading tex-svg-full.js. )

I have discovered an inelegant but functional method. Before loading the MathJax script, I back up window.mathjax, and after loading, I restore it. I rename MathJax in Excalidraw as MyMathJax to avoid conflicts. 


1. rename all 'MathJax' in tex-svg-full.js to 'MyMathJax '
2. `window.MathJax_backup = window.MathJax`
3. `self.mathjax = win.MyMathJax || win.MathJax;  window.MathJax = window.MathJax_backup`
